src/projects.rs

```diff
@@ -67,6 +98,33 @@ fn fetch_next_item(config: &Config, project_name: &str) -> Result<Option<Item>,
     Ok(items.first().map(|item| item.to_owned()))
 }
 
+/// Fetch projects and prompt to add them to config one by one
+pub fn import(config: Config) -> Result<String, String> {
```

Comment:
This function is doing a lot(preparing prompts, filtering projects, iterating over projects, collecting the input, and modifying the config). Maybe we can split it into some logical parts.

Reply:
Absolutely, split out a few things